### ISSUE TITLE
Cannot download all artist work

### ISSUE THREAD

**Issue by pongsatond** [Nandaka/NijieDownloader#54]
![untitled 2](https://user-images.githubusercontent.com/34614637/34077591-0cae26fe-e33a-11e7-91da-e772988e09ef.png)
![untitled](https://user-images.githubusercontent.com/34614637/34077611-6bb51270-e33a-11e7-9210-2c622306521b.png)

**Comment by Nandaka** [comment]
Ensure you are using the latest version.
![image](https://user-images.githubusercontent.com/836867/34086872-5d7b0eda-e3da-11e7-9e1d-d1b72f61dfd1.png)

It is working on my side.
![image](https://user-images.githubusercontent.com/836867/34086883-6f582a84-e3da-11e7-8ac2-07f1c799d4a5.png)

Else, please upload the log file (e.g. log-2017-12-18.txt) in the application folder.

**Comment by pongsatond** [comment]
[log-2560-12-19.txt](https://github.com/Nandaka/NijieDownloader/files/1571326/log-2560-12-19.txt)

**Comment by pongsatond** [comment]
![untitled](https://user-images.githubusercontent.com/34614637/34153550-0f330a78-e4e5-11e7-8da2-c001d0fecc9b.png)

**Comment by Nandaka** [comment]
Hmm, it shows the work date is in wrong format.

Can you check if the website shows like below format? Mine is set in japanese.
![image](https://user-images.githubusercontent.com/836867/34202733-9f27b600-e5b3-11e7-93b3-aef74ee40678.png)

Also shows me the settings in the Download tab.
![image](https://user-images.githubusercontent.com/836867/34202863-06af1ba6-e5b4-11e7-8f30-0162294e5f83.png)

**Comment by pongsatond** [comment]
![untitled1](https://user-images.githubusercontent.com/34614637/34285452-5ca42944-e70c-11e7-800c-a2213c665241.png)
![untitled](https://user-images.githubusercontent.com/34614637/34285453-5cdc2f60-e70c-11e7-9cad-ef3a3e12359b.png)
[log-2560-12-22.txt](https://github.com/Nandaka/NijieDownloader/files/1581429/log-2560-12-22.txt)
Do you mean set in Japanese locale?

**Comment by pongsatond** [comment]
Buddhist calendar.

**Comment by Nandaka** [comment]
Not really, you can try to use standard gregorian calendar (e.g. 2017)? The timezone shouldn't be the problem.

**Comment by pongsatond** [comment]
![untitled](https://user-images.githubusercontent.com/34614637/34452672-a0626f4e-ed77-11e7-85dc-dd530d0736fd.png)
Finally. God damn! Such a small little thing to ruin our day. Thanks.
